specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-03-01-preview/EntitiesCommon.json

```diff
@@ -0,0 +1,193 @@
+{
```

Comment:
maybe "EntitiesCommonDefenitions"?

Reply:
mmm..
I like this suggestion, thank you